website/docs/migration_guide.html.markdown

```diff
@@ -0,0 +1,135 @@
+---
+layout: "scaleway"
+page_title: "Migration Guide"
+description: |-
+  Migrating your Scaleway provider from v1 to v2.
+---
+
+# Migrating from v1 to v2
+
+This page guides you through the process of migrating your version 1 resources to their version 2 equivalent.
+In version 2 we completely changed the naming of all resources, as well as their attributes.
+
+## Configuration
+
+To accomodate to the changes in the API and to be future proof, we decided to change the configuration management in version 2.
```

Comment:
Something like:

In order to unify configuration management across all scaleway developer tools, we decided